api/package.json

```diff
@@ -24,8 +24,10 @@
     "luxon": "^3.4.4",
     "markdown-it": "^14.1.0",
     "moment": "^2.30.1",
+    "multer": "^2.0.2",
```

Comment:
Might be better to use `express-fileupload` or `express-form-data` since we are already using them in other projects.